packages/package_info_plus_macos/pubspec.yaml

```diff
@@ -18,6 +18,3 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-
-dev_dependencies:
-  pedantic: ^1.8.0
```

Comment:
is this correct or should it be bumped to 1.9.2?

Reply:
well spotted, actually there's not even dart code on the macos projects so I think the dependency could be removed